src/applications/healthcare/questionnaire/components/veteran-info/utils/index.js

```diff
@@ -0,0 +1,68 @@
+import { states50AndDC } from 'vets-json-schema/dist/constants.json';
+
+const stateCodeToFullState = stateCode => {
+  return states50AndDC.filter(f => f.value === stateCode)[0]?.label;
+};
+
+const addressToDisplay = address => {
+  const rv = [];
```

Comment:
Could we come up with a slightly more descriptive const than `rv`? Not sure about the JS convention here so I'm going to default to your expertise!

Reply:
I like `rv`, but I get the read-ability angle. I will update

Reply:
That really was a nitpick. Sorry :)